languages/ruby/exercises/concept/numbers/.docs/introduction.md

```diff
@@ -0,0 +1,24 @@
+There are two different types of numbers in Ruby:
+
+- Integers: numbers with no digits behind the decimal separator (whole numbers). Examples are `-6`, `0`, `1`, `25`, `976` and `500000`.
+- Floating-point numbers: numbers with zero or more digits behind the decimal separator. Examples are `-2.4`, `0.1`, `3.14`, `16.984025` and `1024.0`.
+
+They are implemented through the `Integer` and `Float` class.
+
+Both instances of `Float` and `Integer` implement methods to convert values from one to the other. As a number implemented through the `Integer` class has less precision than a number implemented through the `Float` class, converting from an instance of `Integer` to an instance of `Float` is safe because it does not result in any kind of data loss. However, converting from an instance of `Float` to an instance of `Integer` could mean losing data.
```

Comment:
```suggestion
The `Float` and `Integer` classes have methods that will coerce values from one to the other.  `Integer` numbers are precise to a whole unit, while `Float` has precision that is fractional to an whole number.  This means that coercing a float to an integer may result in loss of precision.
```
We can't "convert" a number from one type to another, as Ruby is strongly typed.  But Ruby does provide "coercion" methods that allow us to present one type as another.  An example of this are the methods `to_i`, `to_f`, and are similar to what we may have seen if we have already covered `Strings`, with `to_s`.